SwiftCompilerSources/Sources/Optimizer/Analysis/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 # This source file is part of the Swift.org open source project
 #
-# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+# Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
 # Licensed under Apache License v2.0 with Runtime Library Exception
 #
 # See http://swift.org/LICENSE.txt for license information
 # See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 
 swift_compiler_sources(Optimizer
   AliasAnalysis.swift
-  CalleeAnalysis.swift)
+  CalleeAnalysis.swift
+  RCIdentityAnalysis.swift)
```

Comment:
As you said, please remove that when you have your own use-def walk.
Please put your own utility function for that directly into ReleaseDevirtualizer.swift.
As we discussed offline, in future we can use the new EscapeInfo utility to do a better analysis.